hummingbird/ml/_topology.py

```diff
@@ -185,6 +218,77 @@ def fix_graph(graph):
             return num_fixed
 
         fix_graph(hb_model.graph)
+    elif backend == tvm_backend:
+        # First we need to generate the torchscript model.
+        batch_trace_input, remainder_trace_input = _get_trace_input_from_test_input(
+            extra_config[constants.TEST_INPUT], batch_size
+        )
+        ts_model = _jit_model(torch_model, batch_trace_input, "cpu", extra_config)
+        if remainder_trace_input is not None:
+            remainder_ts_model = _jit_model(torch_model, remainder_trace_input, "cpu", extra_config)
+
+        # Generate the test input in the TVM format. In case we have a remainder beyond the batch, generate a remainder test input as well.
+        test_input = [
+            (
+                topology.raw_model.input_names[i],
+                batch_trace_input[i].shape if type(batch_trace_input) is tuple else batch_trace_input.shape,
+            )
+            for i in range(len(topology.raw_model.input_names))
+        ]
+        if remainder_trace_input is not None:
+            remainder_test_input = [
+                (
+                    topology.raw_model.input_names[i],
+                    remainder_trace_input[i].shape if type(remainder_trace_input) is tuple else remainder_trace_input.shape,
+                )
+                for i in range(len(topology.raw_model.input_names))
+            ]
+
+        # Pick the proper target.
+        if device == "cuda":
+            target = tvm.target.cuda()
+            ctx = tvm.gpu()
+        elif device == "cpu":
+            target = "llvm"
+            ctx = tvm.cpu()
+        elif "llvm" in device:
+            target = device
+            ctx = tvm.cpu()
+        else:
+            raise RuntimeError("Device {} not recognized".format(device))
+
+        # Get configuration parameters.
+        config = {}
+        if constants.TVM_MAX_FUSE_DEPTH in extra_config:
+            config["relay.FuseOps.max_depth"] = extra_config[constants.TVM_MAX_FUSE_DEPTH]
+        else:
+            # 50 is a good depth for operator fusion. More than that will probably hurt performance.
```

Comment:
If a depth of more than 50 will hit perf, should we print a warning on line 263, or have some sort of error checking there?  (not sure on this, just an idea)

Reply:
The performance problem is model related. I think we already have a comment on the constant definition in supported. Do you believe we need to add also a warning on top of that?

Reply:
It think it's ok as-is. Seems more of a TVM issue than HB, so no warn seems fine